### PR TITLE
Added New Tool

### PR DESCRIPTION
**Pull request by MiteshShah**
Added AnsiPress tool on Command Line Fu

### CODE REVIEW

**Comment by franz-josef-kaiser** [comment]
Thanks for the PR! Quick question: What is _"AnsiPress tool on Command Line Fu"_?

**Comment by MiteshShah** [comment]
@franz-josef-kaiser AnsiPress = Ansible + WordPress

Which help end user to setup NGINX/PHP7/MariaDB/Google PageSpeed stack for WordPress easily.

AnsiPress can take care of setting up stack packages and install/configure WordPress on the fly!

File: index.md [hunk 1]
```diff
@@ -56,6 +56,7 @@ Base Tools
 
 -   [WP-CLI](https://github.com/wp-cli/wp-cli) - The command-line tool for managing WordPress.
 -   [WP-CLI GUI](https://wpcligui.com/) - A GUI for the command line interface for WordPress.
+-   [AnsiPress] (https://github.com/AnsiPress/AnsiPress) - The command-line tool for manage WordPress website on NGINX/PHP7/MariaDB/Google-PageSped stack.
```
Comment (tfrommen) [inline index.md]: Typos: "...for **managing** WordPress **websites** on .../Google-**PageSpeed** stack."

I find the whole wording a little off, though.

Comment (MiteshShah) [reply index.md]: Oops Fixed

**Comment by MiteshShah** [comment]
@franz-josef-kaiser can you remove extra space between [AnsiPress]  ( 
I guess that extra space break the HTML rendering :(

**Comment by franz-josef-kaiser** [comment]
@MiteshShah Please just update the PR and push again. Thanks!

**Pull request merged**